### PR TITLE
Rename package back to tendermint/tendermint and use replace to point…

### PR DESCRIPTION
**Pull request by lasarojc**
… to the current repo (revert #192 and #156 ). This way the repo can be a real drop-in replacement for TM Core, using `replace`.

```bash
for file in `find . \( ! -regex '.*/\..*' \) -type f \( -name "*.go" -o -name "*.proto" \)  `
do
  sed -i '' "s/cometbft\/cometbft\//tendermint\/tendermint\//g" $file
done

gofmt -l -s -w .
make proto-gen
make mockery
```

---

#### PR checklist

- [ ] Tests written/updated
- [ ] Changelog entry added in `.changelog` (we use [unclog](https://github.com/informalsystems/unclog) to manage our changelog)
- [ ] Updated relevant documentation (`docs/` or `spec/`) and code comments

### CODE REVIEW

**Comment by thanethomson** [comment]
Is it not possible to just revert the commits that introduced the change initially? i.e. by doing a `git revert` from the `v0.34.x` branch of the commits associated with the following PRs and in the following order:

- #192 
- #156 
- #108

(Not sure if there are any other relevant PRs?)

File: go.mod [hunk 1]
```diff
@@ -296,3 +296,5 @@ require (
 	mvdan.cc/lint v0.0.0-20170908181259-adc824a0674b // indirect
 	mvdan.cc/unparam v0.0.0-20220706161116-678bad134442 // indirect
 )
+
+replace github.com/informalsystems/tendermint v0.34.26 => ./
```
Comment (thanethomson) [inline go.mod:300]: Why is this necessary?

Comment (lasarojc) [reply go.mod]: Thanks, @thanethomson
Indeed, not necessary.

File: light/detector.go [hunk 2]
```diff
@@ -245,7 +245,7 @@ func (c *Client) handleConflictingHeaders(
 	if primaryBlock.Commit.Round != witnessTrace[len(witnessTrace)-1].Commit.Round {
 		c.logger.Info("The light client has detected, and prevented, an attempted amnesia attack." +
 			" We think this attack is pretty unlikely, so if you see it, that's interesting to us." +
-			" Can you let us know by opening an issue through https://github.com/cometbft/cometbft/issues/new?")
+			" Can you let us know by opening an issue through https://github.com/tendermint/tendermint/issues/new?")
```
Comment (sergio-mena) [inline light/detector.go:248]: Do we want to revert this message?

File: light/doc.go [hunk 3]
```diff
@@ -94,7 +94,7 @@ Check out other examples in example_test.go
 ## 2. Pure functions to verify a new header (see verifier.go)
 
 Verify function verifies a new header against some trusted header. See
-https://github.com/cometbft/cometbft/blob/v0.34.x/spec/light-client/verification/README.md
+https://github.com/tendermint/tendermint/blob/v0.34.x/spec/light-client/verification/README.md
```
Comment (sergio-mena) [inline light/doc.go:97]: I would not revert this

File: light/doc.go [hunk 4]
```diff
@@ -121,7 +121,7 @@ See
 https://docs.cometbft.com/v0.34/core/light-client.html
 for usage example.
 Or see
-https://github.com/cometbft/cometbft/blob/v0.34.x/spec/light-client/README.md
+https://github.com/tendermint/tendermint/blob/v0.34.x/spec/light-client/README.md
```
Comment (sergio-mena) [inline light/doc.go:124]: I would not revert this

File: rpc/core/doc.go [hunk 5]
```diff
@@ -2,7 +2,7 @@
 Package core defines the CometBFT RPC endpoints.
 
 CometBFT ships with its own JSONRPC library -
-https://github.com/cometbft/cometbft/tree/v0.34.x/rpc/jsonrpc.
+https://github.com/tendermint/tendermint/tree/v0.34.x/rpc/jsonrpc.
```
Comment (sergio-mena) [inline rpc/core/doc.go:5]: Let's not revert this one

File: rpc/jsonrpc/doc.go [hunk 6]
```diff
@@ -80,5 +80,5 @@
 //
 // # Examples
 //
-// - [CometBFT](https://github.com/cometbft/cometbft/blob/v0.34.x/rpc/core/routes.go)
+// - [CometBFT](https://github.com/tendermint/tendermint/blob/v0.34.x/rpc/core/routes.go)
```
Comment (sergio-mena) [inline rpc/jsonrpc/doc.go:83]: Un-revert?

**Review by sergio-mena — APPROVED** [review]
Please take a look at the suggestions above

File: light/doc.go (hunk 3 above)
Comment (lasarojc) [inline light/doc.go:97]:
```suggestion
https://github.com/comet/comet/blob/v0.34.x/spec/light-client/verification/README.md
```

File: light/doc.go (hunk 4 above)
Comment (lasarojc) [inline light/doc.go:124]:
```suggestion
https://github.com/comet/comet/blob/v0.34.x/spec/light-client/README.md
```

File: light/detector.go (hunk 2 above)
Comment (lasarojc) [inline light/detector.go:248]:
```suggestion
			" Can you let us know by opening an issue through https://github.com/comet/comet/issues/new?")
```

File: rpc/jsonrpc/doc.go (hunk 6 above)
Comment (lasarojc) [inline rpc/jsonrpc/doc.go:83]:
```suggestion
// - [CometBFT](https://github.com/comet/comet/blob/v0.34.x/rpc/core/routes.go)
```

File: evidence/doc.go [hunk 7]
```diff
@@ -1,7 +1,7 @@
 /*
 Package evidence handles all evidence storage and gossiping from detection to block proposal.
 For the different types of evidence refer to the `evidence.go` file in the types package
-or https://github.com/cometbft/cometbft/blob/v0.34.x/spec/consensus/light-client/accountability.md.
+or https://github.com/tendermint/tendermint/blob/v0.34.x/spec/consensus/light-client/accountability.md.
```
Comment (lasarojc) [inline evidence/doc.go:4]:
```suggestion
or https://github.com/comet/comet/blob/v0.34.x/spec/consensus/light-client/accountability.md.
```

File: rpc/core/doc.go (hunk 5 above)
Comment (lasarojc) [inline rpc/core/doc.go:5]:
```suggestion
https://github.com/comet/comet/tree/v0.34.x/rpc/jsonrpc.
```
